mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -138,10 +139,8 @@ def _prepare_mrp_move_data_bom_explosion(
             "mrp_origin": "mrp",
             "mrp_order_number": None,
             "parent_product_id": bom.product_id.id,
-            "name": ("Demand Bom Explosion: " + name).replace(
-                "Demand Bom Explosion: Demand Bom " "Explosion: ",
-                "Demand Bom Explosion: ",
-            ),
+            "name": "Demand Bom Explosion: %s"
+            % (name or product.product_id.default_code or product.product_id.name),
```

Comment:
Leave this as it was: because you are provoking this now:
![image](https://user-images.githubusercontent.com/23449160/75428973-5282a600-5949-11ea-8032-8203fefdb617.png)

Reply:
If I keep like it was, when a planned order is created without description fields ( which is not required) an error will be generated.
I can modify code adding previous replace change.